apps/dashboard/lib/ctrl.ts

```diff
@@ -0,0 +1,76 @@
+import { env } from "@/lib/env";
+
+/**
+ * Helper function to make authenticated requests to the ctrl service
+ */
+export async function makeCtrlRequest(
```

Comment:
Don't we have to add this to the 
```
const ctrl = createClient
``` 
in `apps/dashboard/lib/trpc/routers/deploy/deployment/rollback.ts` instead?d

Reply:
yeah probably. I'll put this back in draft until I finish testing the dashboard side. I got ahead of myself when andreas told me to put my PR up